lib/scrabble.rb

```diff
@@ -1,4 +1,13 @@
 module Scrabble
+  require "awesome_print"
+  require "pathname"
+  require "csv"
   # require your gems and classes here
   # require_relative 'scrabble/whatevs'
+
+  PROJECT_ROOT = Pathname(__dir__).parent
+
+  require_relative PROJECT_ROOT.join('lib/scrabble/scrabble')
```

Comment:
I'm taking this as a recommendation to experiment more with paths and with require vs. require_relative until I really understand them! I did this out of paranoia, because my partner and I were having so many problems with path errors in the FarMar project & I was so afraid it would happen again. Like so many problems that we lost an hour. >_>